File: src/main/java/com/springboot/app/model/Employee.java
package com.springboot.app.model;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Data
@Entity
@Table(name = "Employee")
public class Employee {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;
	private String name;
	private String jobTitle;
	private int age;
	private Date startDate;
	private Date endDate;
}

File: src/angular-search/src/app/employee.ts
export class Employee{
    public id: number;
    public name: string;
    public jobTitle: string;
    public age: number;
    public startDate: Date;
    public endDate: Date;

    constructor(id: number, name: string, jobTitle: string, 
        age: number, startDate: Date, endDate: Date){
        this.id = id;
        this.name = name;
        this.jobTitle = jobTitle;
        this.age = age;
        this.startDate = startDate;
        this.endDate = endDate;
    }
}